### PR TITLE
Week 2 changes to Coursera branch

### PR DESCRIPTION
**Pull request by yhn112**
PEP8 and visualization part added

### CODE REVIEW

File: week2_model_based/practice_vi.ipynb [hunk 1]
```diff
@@ -22,12 +24,18 @@
...
    "metadata": {
-    "collapsed": true
+    "ExecuteTime": {
+     "end_time": "2018-04-02T13:44:32.642838Z",
+     "start_time": "2018-04-02T13:44:32.545142Z"
+    }
    },
    "outputs": [],
    "source": [
+    "# If you Colab, uncomment this please\n",
```
Comment (Pastafarianist) [inline week2_model_based/practice_vi.ipynb]: > If you Colab

Is this the same in `master`?

File: week2_model_based/practice_vi.ipynb [hunk 2]
```diff
@@ -200,32 +506,43 @@
...
+    "    diff = max(abs(new_state_values[s] - state_values[s])\n",
+    "               for s in mdp.get_all_states())\n",
     "    print(\"iter %4i   |   diff: %6.5f   |   \" % (i, diff), end=\"\")\n",
-    "    print('   '.join(\"V(%s) = %.3f\" % (s, v) for s, v in state_values.items()), end='\\n\\n')\n",
+    "    print('   '.join(\"V(%s) = %.3f\" % (s, v)\n",
+    "                     for s, v in state_values.items()), end='\\n\\n')\n",
     "    state_values = new_state_values\n",
     "\n",
-    "    if diff < min_difference:\n",
+    "    if dif < min_difference:\n",
```
Comment (Pastafarianist) [inline week2_model_based/practice_vi.ipynb]: This looks like a typo.

**Comment by dniku** [comment]
It remains to include the fixes for the MDP structure in this PR.

**Comment by yhn112** [comment]
#255 should be merged instead of this one

**Pull request closed without merging**